src/utils/ModalWithBackdrop.vue

```diff
@@ -126,16 +126,27 @@ export default {
         min-width: 75vw;
     }
     z-index: $zindex-modal;
+    // For phone we set the width and height fixed to 100% of the view.
+    // dvh takes into account the user interface in mobile browsers (with vh part of the modal is
+    // not visible if ui is shown). Is recognized by browsers from 2022 or newer. If the browser
+    // is older, 90vh will normally be used, which is a bit less clean but good enough.
```

Comment:
Although this is a good idea for the topic use case where the window take anyway the whole screen, this doesn't fit well with all use cases. For the `Delete` and `Share` use case of the drawing the modal is smaller and it doesn't make sense to have it as a banner in the middle. 
![image](https://user-images.githubusercontent.com/67745584/229418956-9f4131ce-69ea-4726-94e0-8561363e5cb0.png)
So I would either left it as before or only make it fullscreen when it make sense. When making it fullscreen the rounded corner needs to be removed though.